packages/react-native/React/Base/macOS/RCTUIKit.m

```diff
@@ -349,6 +350,13 @@ - (void)viewBoundsChanged:(NSNotification*)__unused inNotif
   }
 }
 
+- (BOOL)hasMouseHoverEvent
```

Comment:
`[nit|non-blocking]` it's pretty nitty of me to not be fond of moving this code to a superclass for sharing purposes between 2 of N subclasses of RCTUIView, but I don't think it should present a problem so long as the mouse event properties are isolated to RCTView and RCTTextView